script.js

```diff
@@ -25,3 +26,186 @@ const Contact = document.querySelector('#Contact');
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+  {
+    projectName: 'Tonic',
+    projectDescription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+  {
+    projectName: 'Tonic',
+    projectDescription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+];
+
+const projectBtns = document.querySelectorAll('.see-project');
+Array.from(projectBtns).forEach((projectBtns, i) => {
+  projectBtns.addEventListener('click', () => {
+    const viewportWidth = window.innerWidth || document.documentElement.clientWidth;
+
+    if (viewportWidth >= 768) {
+      const popDiv = document.createElement('div');
+      popDiv.id = 'blur-background';
+      popDiv.classList.add('blur-background');
+      popDiv.style.position = 'fixed';
+      popDiv.style.display = 'block';
+      popDiv.style.top = '0';
+      popDiv.style.left = '0';
+      popDiv.style.width = '100vw';
+      popDiv.style.height = '100vh';
+      popDiv.style.zIndex = '11';
+      popDiv.style.backdropFilter = 'blur(6px)';
+      popDiv.innerHTML = `
+        <div class='d-card'>
+          <div class='D-Pop-Header'>
+            <div>
+              <h3 id='name'>${projectsArr[i].projectName}</h3>
+              <div id='exp' class='exp'>
+                <p>CANOPY</p>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+            </div>
+            <a><button id='desk-popup-close' class='popup-close'>X</button></a>
+          </div>
+
+          <img src='./assets/images/popup-img-desktop.svg'/>
+
```

Comment:
- [ ] Kindly add this image into your array as part of the data to be rendered. This part of the project requireent, and rightly asked. This skill will come in handy when you start to work with APIs in the future 👍 

## Requirements:
![image](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/50990925/8b73b3b3-f910-464b-9106-15bd69c1bb9d)

Reply:
@Shedrack-Sunday Thanks it's now resolved.